### DISCUSSION TITLE
How to merge specific doc in the file

### DISCUSSION THREAD

**Discussion by mkruliv** [mikefarah/yq#1496] [Q&A] [answered]
Hi. Thanks for the great app.
For v3 i had the next script
`yq merge --doc 5 --inplace -x app.yaml folder/app_deploy.yaml`
that was used for replacing 5th doc (deployment) in application Kubernetes  manifest yaml file. Is it possible to do the same with version 4?
I've tried smth like
`yq eval-all -i 'select(fileIndex == 0).kind[]="Deployment" * select(fileIndex == 1)' app.yaml folder/app_deploy.yaml`
but it just adds folder/app_deploy.yaml at the end of app.yaml  file

**Comment by mikefarah** [comment, marked as answer]
Simplest command is something like:

```
yq -i 'select(di==4) *= load("folder/app_deploy.yaml")' folder/app.yaml
```

or if you want to find by kind, it will be something like:

```
yq -i 'select(.kind == "Deployment") *= load("folder/app_deploy.yaml")' folder/app.yaml
```

**Comment by mkruliv** [comment]
Thank you very much. It works as needed
